src/filters/oidc/token_response.cc

```diff
@@ -81,7 +81,7 @@ std::shared_ptr<TokenResponse> TokenResponseParserImpl::Parse(
   const auto status = ::google::protobuf::util::JsonStringToMessage(
       raw_string_piece, &message, options);
   if (!status.ok()) {
-    spdlog::info("{}: JSON parsing error: {}", __func__,
+    spdlog::warn("{}: JSON parsing error: {}", __func__,
```

Comment:
why not `error` and for the rest?

Reply:
My consideration is just "system error" should be `error`. But here is not a system error.

Reply:
My consideration is just severe "system error" that won't be processed anymore should be `error`. But here is not so critical, and no problem to continue entire process.